### DISCUSSION TITLE
Ability to disable world but protect dropped items

### DISCUSSION THREAD

**Discussion by Ergenta** [GriefPrevention/GriefPrevention#630] [Q&A] [answered]
If died and drop items in a grefiprevention disabled world
 Is there a way to still protect the dropped items

**Because** I wanna be the very best, Like no one ever was. Pokemon!

Here's how to increase your chances of your feature request being understood (and potentially considered for implementation): https://content.pivotal.io/blog/how-to-write-well-formed-user-stories

**Comment by Ergenta** [comment]
Or a way to enable the world do the dropped items are protected but disable the claim function. I know if you world guard it, it prevents claims, but that also prevents building too. I have a resource world and I want the items dropped to be protected but disallow the ability to claim

Can you change the label to question? It wasn't an option when I made this

**Comment by Jikoo** [comment, marked as answer]
You can enable claims but negate the permission `griefprevention.claims` for users in that world. This will prevent them creating claims but allow you to use other GP features as you see fit.

**Comment by RoboMWM** [comment]
I think you can set PvP rules for worlds that have disabled claims. I could be wrong but I believe this is the case.

**Comment by Jikoo** [comment]
Closing due to lack of response - assuming one of the two suggestions worked.
